Error in OSPREY3/examples/python.GMEC/findGMEC.cats.py

Running the following code results in this error:
>>>python findGMEC.cats.py
Traceback (most recent call last):
  File "/Applications/osprey-osx-python2-3.2-beta1/examples/python.GMEC/findGMEC.cats.py", line 36, in <module>
    gmec = dgf.calcGMEC()
jpype._jexception.ErrorPyRaisable: java.lang.Error: can't write object

java.lang.Error: can't write object
	at edu.duke.cs.osprey.tools.ObjectIO.writeObject(ObjectIO.java:180)
	at edu.duke.cs.osprey.gmec.PrecomputedMatrices.loadMatrix(PrecomputedMatrices.java:256)
	at edu.duke.cs.osprey.gmec.PrecomputedMatrices.loadEPICMatrix(PrecomputedMatrices.java:236)
	at edu.duke.cs.osprey.gmec.PrecomputedMatrices.precompute(PrecomputedMatrices.java:165)
	at edu.duke.cs.osprey.gmec.PrecomputedMatrices.<init>(PrecomputedMatrices.java:110)
	at edu.duke.cs.osprey.gmec.DEEGMECFinder.calcGMEC(DEEGMECFinder.java:142)
	at edu.duke.cs.osprey.gmec.DEEGMECFinder.calcGMEC(DEEGMECFinder.java:120)
Caused by: java.io.NotSerializableException: edu.duke.cs.osprey.dof.MutAlignmentCache
	at java.base/java.io.ObjectOutputStream.writeObject0(Unknown Source)
	at java.base/java.io.ObjectOutputStream.defaultWriteFields(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeSerialData(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeOrdinaryObject(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeObject0(Unknown Source)
	at java.base/java.io.ObjectOutputStream.defaultWriteFields(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeSerialData(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeOrdinaryObject(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeObject0(Unknown Source)
	at java.base/java.io.ObjectOutputStream.writeObject(Unknown Source)
	at edu.duke.cs.osprey.tools.ObjectIO.writeObject(ObjectIO.java:172)
	... 6 more

The same error occurs in release 3.1 and occurs in both Linux CENTOS 7 and MacOS 10.15 operating systems

--- a/src/edu/duke/cs/osprey/ematrix/epic/EPICSettings.java
+++ b/src/edu/duke/cs/osprey/ematrix/epic/EPICSettings.java
@@ -67,7 +67,7 @@ public class EPICSettings implements Serializable {
     boolean useSAPE = true;
     boolean usePC = true;//use principal-component polynomials
     
-    boolean quadOnly = false;//use only quadratic polynomials + SAPE
+    public boolean quadOnly = false;//use only quadratic polynomials + SAPE
     
     public boolean minPartialConfs = true;//In A*, minimize sum of polynomial fits for partially
     //defined conformations, as well as for fully enumerated ones.  Increase A* lower bounds

